ui/client/components/toolbars/view/buttons/BussinesViewSwitch.tsx

```diff
@@ -12,7 +11,7 @@ import {ReactComponent as Icon} from "../../../../assets/img/toolbarButtons/busi
 type Props = StateProps
 
 function BussinesViewSwitch(props: Props) {
-  const {businessView, businessViewChanged, fetchProcessToDisplay, nothingToSave, processId, versionId} = props
+  const {businessView, fetchProcessToDisplay, nothingToSave, processId, versionId} = props
   const {t} = useTranslation()
   return (
     <ToolbarButton
```

Comment:
Right new we don't need info about business view?

Reply:
we have this info inside process, no need to add it separately and then sync.